source/jormungandr/jormungandr/interfaces/v1/serializer/pt.py

```diff
@@ -440,7 +440,9 @@ class JourneyPatternSerializer(GenericSerializer):
 
 class StopTimeSerializer(PbNestedSerializer):
     arrival_time = LocalTimeField()
+    utc_arrival_time = LocalTimeField()
```

Comment:
`utc_` has a `LocalTimeFiled()` ? :D you may want to give that Field a more generic name

Reply:
It's true, we can change the name of the function "LocalTimeField" in "TimeField", because we use for other fields like (utc, base arrival, amended arrival, etc...).